common/reactive/src/main/java/io/helidon/common/reactive/Multi.java

```diff
@@ -842,10 +855,23 @@
      * Terminal stage, invokes provided consumer for every item in the stream.
      *
      * @param consumer consumer to be invoked for each item
-     */
-    default void forEach(Consumer<? super T> consumer) {
-        FunctionalSubscriber<T> subscriber = new FunctionalSubscriber<>(consumer, null, null, null);
+     * @return MultiCompletionStage
```

Comment:
Wrong javadoc for return type

Reply:
Also the type is clearly visible from Javadoc already, this should describe behavior, not type